test/testenv/util.go

```diff
@@ -551,3 +553,38 @@ func ExecuteCommandOnPod(deployment *Deployment, podName string, stdin string) (
 	logf.Log.Info("Command executed on pod", "pod", podName, "command", command, "stdin", stdin, "stdout", stdout, "stderr", stderr)
 	return stdout, nil
 }
+
+// GetSubDirsOnPod returns subdirectory under given path on the given POD
+func GetSubDirsOnPod(deployment *Deployment, podName string, path string) ([]string, error) {
```

Comment:
Do we need to check if there are files in the directory?  Since the current result would be a mix of files and folders